cmd/spire-server/cli/run/run_test.go

```diff
@@ -495,8 +496,8 @@ func TestNewServerConfig(t *testing.T) {
 				c.Server.SocketPath = "foo"
 			},
 			test: func(t *testing.T, c *server.Config) {
-				require.Equal(t, "foo", c.BindUDSAddress.Name)
-				require.Equal(t, "unix", c.BindUDSAddress.Net)
+				require.Equal(t, "foo", c.BindLocalAddress.(*net.UnixAddr).Name)
+				require.Equal(t, "unix", c.BindLocalAddress.(*net.UnixAddr).Net)
```

Comment:
nit: would be more robust if we asserted the correct values using the net.Addr's `Network()` and `String()` methods